common/lib/protocol-definitions/src/config.ts

```diff
@@ -9,8 +9,27 @@
 // * maxTime(ms) have passed with pending ops to summarize, or
 // * maxOps are waiting to summarize
 // AND
-// * disableSummaries !== true
-export interface ISummaryConfiguration {
+// * state === "enabled"
```

Comment:
I do not think we need to change that file - we only need definitions in container runtime.

Reply:
Wait, so won't we need to define the ISummaryConfiguration in the protocol-definitions ? If so, my change becomes much easier.  This file would be part of a separate PR to bump the version from protocol-definitions. If we will not need it, I will clean up the includes.

Reply:
Yes, we do not need to change protocol. We are defining parallel interfaces and starting to use them in runtime.
Protocol versions will be removed at some point.

There is no way to use protocol versions anyway in a single PR - you would need to first publish pre-release versions of protocol before you can consumer them in client code. So, by definitions they can't be used in same PR.

Reply:
I see. I had assumed the final location had to be on the protocol-definitions. And that I would need to eventually remove the definition I added in the containerruntime and use the protocol-definitions one. So we are good.